### ISSUE TITLE
Training loss goes up and down again

### ISSUE THREAD

**Issue by Vaishnvi** [Twizwei/bddoia_project#3]
Hi @Twizwei ,

Thanks for sharing the repo.

I have been trying to run this on my local machine but was facing batch size issue so I used gradient accumulation with accumulation_steps as 4 and batch_size = 2. But I see training loss isn't quite decreasing.

I have attached the screenshot for training for 35 epochs. Can you please help with this , like is anything going wrong here?

![trainlossgithub](https://user-images.githubusercontent.com/24861262/91768408-2f3bfa80-ebfb-11ea-84cf-ee021e6d0fba.png)

**Comment by Twizwei** [comment]
This problem is probably caused by the untrained Faster RCNN module. Usually, we have to pre-train Faster RCNN model on BDD100K and then adapt its weights to our dataset
